attestation/environment/sensitive_env_vars.go

```diff
@@ -14,9 +14,20 @@
 
 package environment
 
-// sourced from https://github.com/Puliczek/awesome-list-of-secrets-in-environment-variables/blob/main/raw_list.txt
-func DefaultBlockList() map[string]struct{} {
+// DefaultSensitiveEnvList return a list of known sensitive environment keys.
+//
+// Eplicit list sourced from https://github.com/Puliczek/awesome-list-of-secrets-in-environment-variables/blob/main/raw_list.txt
+func DefaultSensitiveEnvList() map[string]struct{} {
 	return map[string]struct{}{
+
+		// Glob pattern list
+		"*_TOKEN":    {},
+		"SECRET_*":   {},
+		"*_API_KEY":  {},
+		"*_PASSWORD": {},
+		"*_JWT":      {},
```

Comment:
We should make it less opinionated on how the env var should start or finish.
If you see, for secret, we are ok with having `SECRET_*`, but for token, we expect a prefix of `*_TOKEN`.

With the (amazing) last changes you added (exclude sensitive key), I think we could make it more strict

 
```suggestion
		"*TOKEN*":    {},
		"*SECRET*":   {},
		"*API_KEY*":  {},
		"*PASSWORD*": {},
		"*JWT*":      {},
```

Reply:
Sounds good! Thanks for that addition.